src/main/java/ChatbotRan/RanUi.java

```diff
@@ -0,0 +1,146 @@
+package ChatbotRan;
+
+import java.util.ArrayList;
+
+/**
+ * Displays chatbot responses to the CLI.
+ */
+public class RanUi {
+    /**
+     * Prints the startup message.
+     */
+    public void greet() {
+        System.out.println("Hello. I am ");
+        String art = "__________                \n" +
+                "\\______   \\_____    ____  \n" +
+                " |       _/\\__  \\  /    \\ \n" +
+                " |    |   \\ / __ \\|   |  \\\n" +
+                " |____|_  /(____  /___|  /\n" +
+                "        \\/      \\/     \\/ ";
+        System.out.println(art);
+        System.out.println("What would you like to do today?");
+    }
+
+    /**
+     * Prints a line to separate the next command.
+     */
+    public void line() {
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * Prints all tasks in the TaskList.
+     *
+     * @param taskList list of tasks
+     */
+    public void printTasks(TaskList taskList) {
+        int size = taskList.size();
+        if (size == 0) {
+            System.out.println("You haven't got any tasks.");
+        } else {
+            for (int i = 0; i < size; i++) {
+                System.out.println("Task " + (i + 1) + ":" + taskList.get(i));
+            }
+        }
+    }
+
+    /**
+     * Prints a generic error message.
+     */
+    public void unknown() {
+
+        System.out.println("I didn't understand that.");
+    }
+
+    /**
+     * Prints response after adding a Task
+     *
+     * @param task task
+     */
+    public void addTask(Task task) {
+        System.out.println("I've added this task to the list: ");
+        System.out.println(task);
+    }
+
+    void printNumber(int size) {
+        if (size == 1) {
+            System.out.println("There is now 1 task in the list");
+        } else {
+            System.out.println("There are now " + size + " tasks in the list");
+        }
+    }
+
+    /**
+     * Prints response after deleting a task.
+     *
+     * @param task task
+     */
+    public void delete(Task task) {
+        System.out.println("I've deleted this task: ");
+        System.out.println(task);
+    }
+
+    /**
+     * Prints the shutdown message.
+     */
+    public void bye() {
+        System.out.println("Goodbye, please return soon.");
+    }
+
+    /**
+     * Prints response after attempting to unmark a task.
+     *
+     * @param completed whether the task was complete
+     */
+    public void unmark(boolean completed) {
```

Comment:
A boolean variable should sound like a boolean. 
```suggestion
    public void unmark(boolean isCompleted) {
```